### ISSUE TITLE
verify-dnssec-zone test fails with named-checkzone 9.16.8

### ISSUE THREAD

**Issue by zeha** [PowerDNS/pdns#9660]
- Program: Authoritative
 - Issue type: Bug report

### Short description

regression-tests fail with:

```
verify-dnssec-zone

 --- named-checkzone delegated.dnssec-parent.com
-zone delegated.dnssec-parent.com/IN: loaded serial 2005092501 (DNSSEC signed)
-OK
+tmp.QkouBmQDir:22: DS record at top of zone (delegated.dnssec-parent.com)
+zone delegated.dnssec-parent.com/IN: loading from master file tmp.QkouBmQDir failed: at top of zone
+zone delegated.dnssec-parent.com/IN: not loaded due to errors.
 RETVAL: 0
```

### Environment

 - Operating system: macOS
 - Software version: master 47528f06b54df1437badd58ffe9d8026ce0b8afa
 - Software source: git

```
named-checkzone -v
9.16.8
```

### Steps to reproduce

1. `./start-test-stop 5300 gmysql-both` or `./start-test-stop 5300 gsqlite3-nsec` 

### Expected behaviour

tests should pass ;)

**Comment by zeha** [comment]
https://gitlab.isc.org/isc-projects/bind9/-/issues/1798

"thanks @habbie" ;)

**Comment by mind04** [comment]
Fixed in #10470
